### PR TITLE
Back port CVE fixes for release-0.29

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

**Update Vendor packages for CVE fixes and enhacements** 

image | old | new
-- | -- | --
ap-registry|3.15.4-1|3.16.1
ap-db-bootstrapper | 0.26.7 | 0.26.8
ap-cli-install | 0.26.6 | 0.26.7
ap-base | 3.16.0 | 3.16.1
ap-curator| 5.8.4-15 |  5.8.4-17 
ap-nginx-es | 1.23.0 | 1.23.1
ap-openresty | 1.21.4 |  1.21.4-1
ap-awsesproxy  | 1.3-5 |  1.3-6
ap-fluentd|1.14.6-1|1.15.0
ap-kube-state| 1.9.7 | 2.5.0
ap-nats-server| 2.8.1 | 2.8.1-1
ap-nats-exporter | 0.9.3 | 0.9.3-1
ap-nats-streaming| 0.24.5 | 0.24.5-1
ap-nginx| 1.2.1 | 1.3.0
ap-default-backend | 0.28.7 | 0.28.8
ap-blackbox-exporter | 0.21.1 | 0.21.1-1
ap-auth-sidecar | 1.23.0 |  1.23.1 
ap-vector|0.22.3|0.23.0

## Related Issues

https://github.com/astronomer/issues/issues/4876

## Testing

Do not merge this PR until this text is replaced with details about how these changes were tested.

## Merging

Do not merge this PR until it lists which release branches this PR should be merged / cherry-picked into.

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

**Comment by vishwas-astro** [comment]
@pgvishnuram can you create a GH issue for this PR. Tag it with 0.29.3 label and list out any specific testing QA needs to do as part of it.

File: tests/chart_tests/test_houston_configmap.py [hunk 1]
```diff
@@ -211,13 +223,15 @@ def test_houston_configmap_with_loggingsidecar_customConfig_enabled():
     """Validate the houston configmap and its embedded data with loggingSidecar customConfig Enabled."""
     sidecar_container_name = "sidecar-log-test"
     terminationEndpoint = "http://localhost:8000/quitquitquit"
+    image_name = "quay.io/astronomer/ap-vector:0.22.3"
```
Comment (vishwas-astro) [inline tests/chart_tests/test_houston_configmap.py:226]: should this be 0.23.0?

File: tests/chart_tests/test_houston_configmap.py [hunk 2]
```diff
@@ -175,10 +182,15 @@ def test_houston_configmap_with_loggingsidecar_enabled_with_overrides():
     """Validate the houston configmap and its embedded data with loggingSidecar."""
     sidecar_container_name = "sidecar-log-test"
     terminationEndpoint = "http://localhost:8000/quitquitquit"
+    image_name = "quay.io/astronomer/ap-vector:0.22.3"
```
Comment (vishwas-astro) [inline tests/chart_tests/test_houston_configmap.py:185]: should this be 0.23.0?

File: tests/chart_tests/test_houston_configmap.py (hunk 1 above)
Comment (pgvishnuram) [reply tests/chart_tests/test_houston_configmap.py:226]: this is a static test case @vishwas-astro

File: tests/chart_tests/test_houston_configmap.py (hunk 2 above)
Comment (pgvishnuram) [reply tests/chart_tests/test_houston_configmap.py:185]: this is a static test case @vishwas-astro

File: values.yaml [hunk 3]
```diff
@@ -105,15 +105,15 @@ global:
   loggingSidecar:
     enabled: false
     name: sidecar-log-consumer
-    image: quay.io/astronomer/ap-vector:0.22.3
+    image: quay.io/astronomer/ap-vector:0.23.0
```
Comment (pgvishnuram) [inline values.yaml:108]: @vishwas-astro actual version is mentioned here

File: charts/nginx/templates/nginx-config-role.yaml [hunk 4]
```diff
@@ -44,4 +44,26 @@ rules:
...
+      - leases
+    verbs:
+      - create
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
```
Comment (vishwas-astro) [inline charts/nginx/templates/nginx-config-role.yaml:68]: Doesn't this make 1.19 of K8s unsupported. That means we are no longer supporting 1.19 on our 0.29.x release?

Comment (pgvishnuram) [reply charts/nginx/templates/nginx-config-role.yaml:68]: removing support only for 1.19.0

**Review by danielhoherd — APPROVED** [review]

**Pull request merged**